tweakreg failure with astropy master

Issue [JP-961](https://jira.stsci.edu/browse/JP-961) was created by Nadia Dencheva:

Note this is in astropy master and the regression test is in JWST_UNSTABLE

"make_tweakreg_catalog" fails with
{code:java}
>       threshold_img = detect_threshold(model.data, snr=snr_threshold)
E       TypeError: detect_threshold() got an unexpected keyword argument 'snr'

{code}
https://plwishmaster.stsci.edu:8081/job/RT/job/JWST_UNSTABLE/22/testReport/junit/jwst.tests_nightly.general.nircam.test_nrc_image3_1/TestImage3Pipeline1/_unstable_deps__test_image3_pipeline2/

--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -146,6 +146,8 @@ tweakreg
 - removed original ``jwst.tweakreg`` alignment code and changed step's code
   to call similar functionality from ``tweakwcs`` package. [#3689]
 
+- Fix deprecated call to photutils.detect_threshold [#3982]
+
 
 0.13.7 (2019-06-21)
 ===================

--- a/jwst/tweakreg/tweakreg_catalog.py
+++ b/jwst/tweakreg/tweakreg_catalog.py
@@ -63,7 +63,7 @@ def make_tweakreg_catalog(model, kernel_fwhm, snr_threshold, sharplo=0.2,
     if not isinstance(model, ImageModel):
         raise TypeError('The input model must be an ImageModel.')
 
-    threshold_img = detect_threshold(model.data, snr=snr_threshold)
+    threshold_img = detect_threshold(model.data, nsigma=snr_threshold)
     # TODO:  use threshold image based on error array
     threshold = threshold_img[0, 0]     # constant image
 

